### PR TITLE
Blog: Baseline Scan Changes

### PR DESCRIPTION
**Pull request by psiinon**
This PR also adds the terminal display option first included in #89

Planned for release on Monday 14th assuming the weekly includes the relevant changes. It will need updating with the latest output.

Screenshot 2021-06-09 at 12-38-19 Baseline Scan Changes

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by psiinon** [comment]
Updated to reflect the latest baseline changes.
Now looks like:
Screenshot 2021-06-10 at 14-36-40 Baseline Scan Changes

**Comment by psiinon** [comment]
Still WIP until weekly release but can be reviewed any time...

**Comment by psiinon** [comment]
Added links to the issues and change the date to today.

**Comment by psiinon** [comment]
Also updated the baseline page with the new options and a note about the AF migration.

**Review by thc202 — APPROVED** [review]

**Comment by psiinon** [comment]
No longer WIP to ready to check nd hopefully publish :)

**Comment by psiinon** [comment]
@kingthorin :)

**Comment by kingthorin** [comment]
Will do it 10-15 when I'm on lunch 👍

File: site/content/blog/2021-06-15-baseline-scan-changes/index.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+	https://www.example.com/robots.txt (404 Not Found)
+	https://www.example.com/ (200 OK)
+	https://www.example.com/sitemap.xml (404 Not Found)
+WARN-NEW: Content Security Policy (CSP) Header Not Set [10038] x 3 
+	https://www.example.com/ (200 OK)
+	https://www.example.com/robots.txt (404 Not Found)
+	https://www.example.com/sitemap.xml (404 Not Found)
+FAIL-NEW: 0	FAIL-INPROG: 0	WARN-NEW: 6	WARN-INPROG: 0	INFO: 0	IGNORE: 0	PASS: 47{{< /terminal >}}
+
+As you will see, in this case the AF reports one more issue that the non AF case.
```
Comment (kingthorin) [inline site/content/blog/2021-06-15-baseline-scan-changes/index.md:261]:
```suggestion
As you will see, in this case the AF reports one more issue that the non-AF case.
```

**Review by kingthorin — COMMENTED** [review]
Just one minor item.

**Comment by kingthorin** [comment]
Feel free to merge after that one thing, even if I don't get back for another look.

**Review by ricekot — APPROVED** [review]
Thank you!

**Comment by psiinon** [comment]
Fixed all of thoe :)

**Pull request merged**